### PR TITLE
add option (-U, --no-unicode) to disable unicode at runtime

### PR DESCRIPTION
**Pull request by eworm-de**

### CODE REVIEW

File: CRT.c [hunk 1]
```diff
@@ -763,12 +763,15 @@ void CRT_init(int delay, int colorScheme) {
    setlocale(LC_CTYPE, "");
 
 #ifdef HAVE_LIBNCURSESW
-   if(strcmp(nl_langinfo(CODESET), "UTF-8") == 0) {
+   if (allowUnicode == true &&
```
Comment (cgzones) [inline CRT.c:766]: `allowUnicode == true` seems a bit redundant

**Review by cgzones — COMMENTED** [review]
LGTM

File: CRT.c (hunk 1 above)
Comment (eworm-de) [reply CRT.c:766]: I can remove that if you prefer...

**Comment by Ckath** [comment]
doesnt seem needed anymore after #54, which I think would be the preferable solution

**Comment by bertwesarg** [comment]
> doesnt seem needed anymore after #54, which I think would be the preferable solution

I think they match up perfectly. Disabling Unicode, even if the terminal is capable, looks like a valid case.

**Comment by eworm-de** [comment]
Let me explain my use case: The graph meters use braille characters to display. This works fine if braille characters are available (X terminal), but fails if not: Only squares are displayed then. I use this to switch to ASCII mode in virtual (non-X) terminal. UTF-8 is supported there, just glyphs are missing.

**Review by natoscott — CHANGES_REQUESTED** [review]
This needs a man page update I think...?  Thanks.

**Comment by eworm-de** [comment]
Ah, right... Will add that.

**Comment by eworm-de** [comment]
Rebased and added the man page change.

**Review by natoscott — COMMENTED** [review]
Thanks - another small thing, it looks like all the #ifdef conditionals in htop.c could be removed around the new code?  The default behaviour without HAVE_LIBNCURSESW is to not 'allowUnicode', right?  - it seems that code can be simplified further, always allowing the -U/--no-unicode option to be entered by the user, no matter how htop is built.

**Comment by eworm-de** [comment]
Probably you are right. :)
Updated (and rebased) again.

**Pull request merged**